libr/anal/var.c

```diff
@@ -959,7 +987,9 @@ R_API void r_anal_var_list_show(RAnal *anal, RAnalFunction *fcn, int kind, int m
 		}
 	}
 	if (mode == 'j') {
-		anal->cb_printf ("]");
+		pj_end (pj);
+		anal->cb_printf ("%s\n", pj_string (pj));
```

Comment:
probably want to use println as pancake said already

Reply:
Yea sure

Reply:
println isn't available in anal->cb_ right?

Reply:
dunno, @radare ?

Reply:
nah, this is fine, noneed to expose more callbacks